compliance/collection/compliance/compliance.go

```diff
@@ -102,6 +109,29 @@ func (c *Compliance) Start() {
 	log.Info("Successfully closed Sensor communication")
 }
 
+// TODO(ROX-24117): This report should be sent to central for enrichment
+func (c *Compliance) manageNodeIndexLoop(ctx context.Context) {
+	go func() {
+		i := c.nodeIndexer.GetIntervals()
+		t := time.NewTicker(i.Initial())
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case <-t.C:
+				log.Infof("Creating Node Index Report")
+				report, err := c.nodeIndexer.IndexNode(ctx)
```

Comment:
This report is created and not sent - is this intentional?

Reply:
Yes, this will be addressed in [ROX-24117](https://issues.redhat.com/browse/ROX-24117).

Reply:
Have a look at the draft PR https://github.com/stackrox/stackrox/pull/12369 which is my work implementing the dataflow towards central